airbyte-integrations/connectors/destination-mssql-v2/src/main/kotlin/io/airbyte/integrations/destination/mssql/v2/MSSQLStreamLoader.kt

```diff
@@ -24,6 +25,13 @@ class MSSQLStreamLoader(
         ensureTableExists(dataSource)
     }
 
+    override suspend fun close(streamFailure: StreamProcessingFailed?) {
+        if (streamFailure == null) {
```

Comment:
Nit:   more Kotlin way would be to do `streamFailure?.let { ... }`

Reply:
I need to enter that block when it's actually null. `?.let {}` is the opposite